packages/shared/routes/setup/ledger/views/Connect.svelte

```diff
@@ -34,7 +34,7 @@
     $: isConnected = $ledgerDeviceState !== LedgerDeviceState.NotDetected
     $: isAppOpen = $ledgerDeviceState === LedgerDeviceState.Connected
 
-    $: illustration = isConnected && isAppOpen ? 'ledger-connected-desktop' : 'ledger-disconnected-desktop'
+    $: illustration = isConnected && isAppOpen ? 'ledger-connect-connected-desktop' : 'ledger-connect-disconnected-desktop'
```

Comment:
@maxwellmattryan perhaps this was a wrong merge?

Reply:
Yes it definitely was, my mistake! I'm making some changes right now fixing that and some locale stuff. After that it should be ready 😅